include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -1063,7 +1075,14 @@ Matrix<T, Device::CPU> ReductionToBand<B, D, T>::call(Matrix<T, D>& mat_a, const
     // TODO probably the first one in any panel is ok?
```

Comment:
Random thought: just saw this comment and wondering if we should have a look and see if we can "merge" this workspace into the other one? @rasolca

Reply:
I will leave it for another PR 😉